ace.d.ts

```diff
@@ -753,3 +753,14 @@ declare namespace Ace {
 }
 
 declare const ace: Ace.AceStatic;
+export default ace;
```

Comment:
i think we could have a fiddle around with [this export pattern](https://github.com/zspitz/TypeScript-Handbook/blob/release-2.0/pages/Modules.md#umd-modules) here. to avoid the duplicated constructors. ill have a go tomorrow